scan/spec/test_command_generator_spec.rb

```diff
@@ -1,5 +1,8 @@
 describe Scan do
+  require "rspec/mocks/standalone"
```

Comment:
Got it. This is pretty unusual, so putting a comment before the `require` to explain why its necessary would be good for our future selves 👍